src/Event/EventSubscriber/ResetTeamItemsOwnerSubscriber.php

```diff
@@ -33,6 +34,20 @@ public function preRemove(GenericEvent $event): void
             return;
         }
 
-        $this->repository->resetOwnerTeamItems($user);
+        foreach ($user->getUserTeams() as $userTeam) {
+            $team = $userTeam->getTeam();
+            $admins = $userTeam->getTeam()->getAdminUserTeams([$user->getId()->toString()]);
+            if (0 === count($admins)) {
+                /** @var UserTeam $nextUserTeam */
+                $nextUserTeam = $team->getUserTeams()->first();
```

Comment:
Maybe it will be better to change name of this variable. Now it is not clear why next UserTeam is the first team from the list